packages/polaris-core/src/config/create-apollo-config-util.ts

```diff
@@ -111,7 +113,12 @@ export function createPolarisSchemaWithMiddlewares(
 ) {
     applyMiddleware(
         schema,
-        new SnapshotMiddleware(config.logger, config.snapshotConfig).getMiddleware(),
+        new SnapshotMiddleware(
```

Comment:
i think that its more elegant to send the config only and inside the method use the relevant properties of the config instead of send them separately